fhir-persistence-schema/docs/tests/Db2.md

```diff
@@ -0,0 +1,217 @@
+2020-05-20 13:17:32.943 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.JOBINSTANCE TO FHIRSERVER
+2020-05-20 13:17:32.969 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.JOBEXECUTION TO FHIRSERVER
+2020-05-20 13:17:32.992 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.STEPTHREADEXECUTION TO FHIRSERVER
+2020-05-20 13:17:33.017 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.STEPTHREADINSTANCE TO FHIRSERVER
+2020-05-20 13:17:33.046 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.REMOTABLEPARTITION_TABLE TO FHIRSERVER
+2020-05-20 13:17:33.079 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.GROUPASSOCIATION TO FHIRSERVER
+2020-05-20 13:17:33.105 00000001    INFO s.common.CommonDatabaseAdapter Applying: GRANT SELECT,INSERT,UPDATE,DELETE ON FHIR_JBATCH.JOBPARAMETER TO FHIRSERVER
+2020-05-20 13:17:33.140 00000001    INFO   com.ibm.fhir.schema.app.Main Processing took:  35.505 s
+2020-05-20 13:17:33.140 00000001    INFO   com.ibm.fhir.schema.app.Main SCHEMA CHANGE: OK
+
+[db2inst1@4dda34a66a99 ~]$ db2 "SELECT DISTINCT TRIM(GRANTEE || ' '), SELECTAUTH|| UPDATEAUTH||DELETEAUTH||INSERTAUTH,CONTROLAUTH||ALTERAUTH||  INDEXAUTH||  REFAUTH  FROM SYSCAT.TABAUTH WHERE TABSCHEMA IN ('FHIRDATA','FHIR_OAUTH','FHIR_JBATCH')"
+1                                                                                                                                 2    3   
+--------------------------------------------------------------------------------------------------------------------------------- ---- ----
+FHIRSERVER                                                                                                                        YYYY NNNN
+DB2INST1                                                                                                                          GGGG YGGG
+
+  2 record(s) selected.
+
+# Test 13 - Allocate Tenant - FHIRDATA
+--prop-file fhir-persistence-schema/db2.properties
+--schema-name FHIRDATA
+--allocate-tenant default
+2020-05-20 13:26:22.393 00000001    INFO   com.ibm.fhir.schema.app.Main tenantId [1] is being pre-populated with lookup table data.
+2020-05-20 13:26:22.606 00000001    INFO   com.ibm.fhir.schema.app.Main Finished prepopulating the resource type and search parameter code/name tables tables
+2020-05-20 13:26:22.649 00000001    INFO   com.ibm.fhir.schema.app.Main Allocated tenant: default [key=DOd7xmwhkx2FhJMyPLwaHeU8PAUWcpGj7VjYgMf2hy0=] with Id = 1
+2020-05-20 13:26:22.651 00000001    INFO   com.ibm.fhir.schema.app.Main Processing took: 270.022 s
+2020-05-20 13:26:22.651 00000001    INFO   com.ibm.fhir.schema.app.Main SCHEMA CHANGE: OK
```

Comment:
postgresql.md is empty, is it the same as db2.md? if so, maybe should not name as db2.md ...

Reply:
intentional (I hadn't yet finished the tests)
it'll be uploaded, thanks for noticing.